server/controllers/users.js

```diff
@@ -96,3 +96,46 @@ export function getAllUsers(req, res, next) {
       console.log('could not retrieve users', err);
     });
 }
+
+function toCamelCase(word) {
+  const wordArr = word.split('_').join('');
+  const camelized = wordArr.slice(0, 2) + wordArr[2].toUpperCase() + wordArr.slice(3);
+  return camelized;
+}
+
+function addIfExists(paramObj, key, status) {
+  if (key) {
+    const newParamObj = paramObj;
+    newParamObj[toCamelCase(key)] = JSON.parse(status);
+  }
+}
+
+export function manageUserById(req, res, next) {
+  if (req.body.isBanned === 'true') {
+    req.body.isActive = 'false';
+  }
+
+  const params = {
+    id: req.params.id
+  };
+  //  This is necessary because params keys are camel cased and the req.body is coming in as snake
+  Object.keys(req.body).forEach((key) => {
```

Comment:
If you're doing a forEach over the keys of req.body, why do you need to check if it exists? Why not just directly parse the k-v pair to camelcase key and JSON-parsed value?

Reply:
Oh right, since this is the helper function, if it exists, it'll get modified. The check is extraneous.